pandas/tests/series/test_operators.py

```diff
@@ -1857,3 +1857,58 @@ def test_op_duplicate_index(self):
         result = s1 + s2
         expected = pd.Series([11, 12, np.nan], index=[1, 1, 2])
         assert_series_equal(result, expected)
+
+    def test_argminmax(self):
+        # Series.argmin, Series.argmax are aliased to Series.idxmin,
+        # Series.idxmax
+
+        # Expected behavior for empty Series
+        s = pd.Series([])
+
+        with pytest.raises(ValueError):
+            s.argmin()
+        with pytest.raises(ValueError):
+            s.argmin(skipna=False)
+        with pytest.raises(ValueError):
+            s.argmax()
+        with pytest.raises(ValueError):
+            s.argmax(skipna=False)
+
+        # For numeric data with NA and Inf (GH #13595)
+        s = pd.Series([0, -np.inf, np.inf, np.nan])
+
+        assert s.argmin() == 1
+        assert np.isnan(s.argmin(skipna=False))
+
+        assert s.argmax() == 2
+        assert np.isnan(s.argmax(skipna=False))
+
+        # Using old-style behavior that treats floating point nan, -inf, and
+        # +inf as missing
+        s = pd.Series([0, -np.inf, np.inf, np.nan])
+
+        with pd.option_context('mode.use_inf_as_na', True):
+            assert s.argmin() == 0
+            assert np.isnan(s.argmin(skipna=False))
+            assert s.argmax() == 0
+            np.isnan(s.argmax(skipna=False))
+
```

Comment:
put these error ones in a separate test function, use parametrize on it to avoid the boilerplate.